### DISCUSSION TITLE
Where can I find MSW 1.x docs?

### DISCUSSION THREAD

**Discussion by Coding-Journey** [mswjs/mswjs.io#263] [Q&A] [answered]
- [Open page](https://mswjs.io/docs)

## Feedback

What happened to your documentation for older versions? How can I access older versions and see the api/examples please?

**Comment by kettanaito** [comment, marked as answer]
Hi, @Coding-Journey. You can find the old docs at https://v1.mswjs.io. I really need to add a banner to the new docs about this.
